add emergency on google storage limit (#1249)
utelecon/utelecon.github.io: src/emergencies/20241101EnGoogleStorageLimit.md

---
pattern: "^/en(/|/support/|/google/.*)$"
---

<div class="box--alert">

>   Currently, there is a problem in which the service on ECCS Cloud Email displays the message “Storage is full,” “Organization storage full,” “The file is view-only” or “You're out of storage,” and prevents writing operations. The main effects are as follows
>   - Unable to upload files to Google Drive
>   - Cannot edit Google Docs / Google Spreadsheets / Google Slides
>   - Google Photos does not work.

Reviewer: ```suggestion
Currently, there is an issue with the service on ECCS Cloud Email displaying messages such as “Storage is full,” “Organization storage full,” “The file is view-only” or “You're out of storage,” which prevents write operations. The main impacts are as follows:

- Unable to upload files to Google Drive
- Unable to edit Google Docs / Google Spreadsheets / Google Slides
- Google Photos not functioning

```

Author: ぶっちゃけどうでもいいので省略
